Framework/cfg/sampleCollections_v6.cfg

```diff
@@ -13,6 +13,12 @@
 # ********************
 
 2016_TT, 2016_TT
+2016_TT_erdON, 2016_TT_erdON
+2016_TT_hdampUP, 2016_TT_hdampUP
```

Comment:
Ok since you need to update this one more time lets change all "UP" to "Up" and all "DOWN" to "Down" so it will match the naming convention later on.